.travis.yml

```diff
@@ -3,7 +3,7 @@ branches:
     - master
 language: objective-c
 os: osx
-osx_image: xcode8
+osx_image: xcode9
```

Comment:
Mistake on the commit message saying "Use Xcode 8" when it should have said "Use Xcode 9"